src/dotnet/Chat.UI.Blazor/Components/ChatList/ChatListSkeleton.razor

```diff
@@ -0,0 +1,12 @@
+@namespace ActualChat.Chat.UI.Blazor.Components
+
+@{
+    var rnd = new Random(Random.Shared.Next());
+    var count = 3 + rnd.Next(6);
```

Comment:
why random?

Reply:
It can be fixed. But with random looks mone intresting, IMHO.